src/shout_subgroup/repository.py

```diff
@@ -137,7 +137,7 @@ async def insert_user(
         last_name=last_name
     )
     db.add(new_user)
-    db.commit()
+    db.flush()
```

Comment:
B/c we're handling the `.commit()` implicitly within the `with` block, calling it here prematurely stopped the session and the subsequent db calls failed. 

Also, we have to use `.flush()` when after .add(). TBH, I don't fully understand why.
https://docs.sqlalchemy.org/en/20/orm/session_basics.html#session-flushing